apps/coscrad-frontend/src/store/slices/utils/buildReducersForThunk.ts

```diff
@@ -21,8 +21,10 @@ export const buildReducersForThunk = <TState extends ILoadable<unknown>, URespon
     // @ts-expect-error fix types
     builder.addCase(thunk.rejected, (state: TState, action) => {
         if (action.payload) {
```

Comment:
It occurs to me that we might want to unit test just one such reducer if for no other reason than to serve as a specification of the expected state-transitions.

Reply:
I've added a story for this [here](https://www.pivotaltracker.com/story/show/183783973).